scripts/receipt_mcp_server.py

```
@@ -1877,6 +1929,54 @@ async def create_word_label_impl(
+
+        response_payload = json.loads(response["Payload"].read())
+
+        # Check for Lambda-level errors
+        if "FunctionError" in response:
+            return {
+                "error": "Lambda execution failed",
+                "function_error": response["FunctionError"],
+                "details": response_payload,
+            }
+
+        return response_payload
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)